### DISCUSSION TITLE
FSunit

### DISCUSSION THREAD

**Discussion by Aryan-sharma11** [Aryan-sharma11/AddSevenPS#12] [General]
- Supports XUnit as well as NUnit 
- Provides more programmer-friendly operators ( should, equal, equivalent )

**Comment by Aryan-sharma11** [comment]
> * Supports XUnit as well as NUnit
> * Provides more programmer-friendly operators ( should, equal, equivalent )

Tried integrating Expecto with Xunit, but it is throwing  "No test available error"

**Comment by Aryan-sharma11** [comment]
@abhi18av  are FSUnit tests still relevant to the project after adding pester tests?

**Reply by abhi18av** [reply]
@Aryan-sharma11 , I think that this has been covered a couple of times in our meetings that we need to think in terms of layers.

1. F# layer => FSUnit  (core library and logic tests)
2. Powershell => Pester (consumer lever testing)

Makes sense?
